README.md

```diff
@@ -253,7 +253,7 @@ The `lib/` folder contains all jars and library files needed to run your test -
  1. Run `./mvnw clean package` to build the jGrade2 jar
  2. Copy the jGrade2 jar to the `lib/` folder
     ```
-    cp target/jgrade2-2.0.0-a2.jar examples/gradescope/lib
+    cp target/jgrade2-2.0.0-a2-all.jar examples/gradescope/lib
```

Comment:
For the longevity of the readme, I would suggest making this command a more template form. Something like:

```
cp target/jgrade2-{version}.jar path/to/your/java/project/lib
```

Or maybe specify that this is an example:

2. Copy the jGrade2 jar to the `lib/` folder
	- Example:
	```
	cp target/jgrade2-2.0.0-a2-all.jar examples/gradescope/lib
	```

Reply:
Complete with [a10e37b](https://github.com/jgrade2/jgrade2/pull/1/commits/a10e37bf1fdc18943c5fe23a1d78b1a545053d4a)